src/platform/site-wide/ebenefits/containers/ebenefitsLink.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { isLoggedIn } from 'platform/user/selectors';
+import recordEvent from 'platform/monitoring/record-event';
+
+import { shouldUseProxyUrl } from '../selectors';
+import { proxyUrl, defaultUrl } from '../utilities';
+
+class ebenefitsLink extends React.Component {
+  static propTypes = {
+    path: PropTypes.string,
+    emitEvent: PropTypes.bool,
+  };
+  static defaultProps = {
+    path: '',
+    emitEvent: true,
+  };
+
+  render() {
+    const url = this.props.useProxyUrl ? proxyUrl : defaultUrl;
+    const defaults = {
+      href: url(this.props.path),
+      target: '_blank',
+      rel: 'noopener noreferrer',
+      onClick: () =>
+        this.props.emitEvent &&
```

Comment:
is this check necessary? I didn't see this getting passed in anywhere

Reply:
I was thinking of adding the extra property so it would be easy for the caller to turn off the event, but I guess thats not really needed because the alternative syntax is pretty simple.  I'll remove it.

`<EbenefitsLink emitEvent={false} />`
or
`<EbenefitsLink onClick={() => {}} />`